force-app/main/default/classes/ServiceDeliveryController.cls

```diff
@@ -33,6 +33,20 @@ public with sharing class ServiceDeliveryController {
         return true;
     }
 
+    @AuraEnabled
```

Comment:
I was expecting to see an Apex Test update in this PR based on this change.  Same with the change to the domain class.